ostd/src/mm/page_table/test.rs

```diff
@@ -61,21 +84,30 @@ fn test_untracked_map_unmap() {
     }
 
     let unmap = UNTRACKED_OFFSET + PAGE_SIZE * 13456..UNTRACKED_OFFSET + PAGE_SIZE * 15678;
-    assert!(matches!(
-        unsafe { pt.cursor_mut(&unmap).unwrap().take_next(unmap.len()) },
-        PageTableItem::MappedUntracked { .. }
-    ));
+
+    let mut cursor = pt.cursor_mut(&unmap).unwrap();
+    assert_eq!(cursor.virt_addr(), unmap.start);
+    let unmapped = unsafe { cursor.take_next(unmap.len()) };
+    let PageTableItem::MappedUntracked { va, pa, len, prop } = unmapped else {
```

Comment:
Is it possible to add a helper method to assert `PageTableItem::MappedUntracked`? It seems like there is only one use for it at the moment, but I think it will lead to better consistency and also make it easier for others to add more unit tests.

(By the way, the commit message should say to add "unit tests" instead of "regression tests").